chain/testing.go

```diff
@@ -56,16 +68,36 @@ func (f *Builder) AppendOn(parents ...*types.Block) *types.Block {
 	if len(parents) > 0 {
 		tip = types.RequireNewTipSet(f.t, parents...)
 	}
-	return f.BuildOnTip(tip, nil)
+	return f.AppendOnTip(tip)
+}
+
+// AppendOnTip creates and returns a new single-block tipset child of `parent`, with no messages.
```

Comment:
This method returns a block, not a tipset. Is this comment a typo or should the method declaration be changed? I don't like the idea of calling a `types.Block` a single-block tipset since it cannot be treated as a `types.Tipset`.